tests/unit/test_index.py

```diff
@@ -308,7 +308,10 @@ def test_request_retries(caplog):
         # URL with something that looks like a drive letter, but is
         # not. The `:` should be quoted.
         ("https://localhost.localdomain/T:/path/",
-         "https://localhost.localdomain/T%3A/path/")
+         "https://localhost.localdomain/T%3A/path/"),
+        # URL with link to git.
```

Comment:
I would say "VCS url containing revision string" since that it is the key thing being tested here.

Reply:
Done